src/safe_apps/migrations/0013_alter_safeapp_icon_url.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 5.0.6 on 2024-05-31 14:46
+# Modified to update existing entries with no icon_url
+
+from django.db import migrations, models
+from django.db.migrations.state import StateApps
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+import safe_apps.models
+
+
+def add_missing_default_icon_url(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    SafeApp = apps.get_model("safe_apps", "SafeApp")
+    default_icon_url = "safe_apps/icon_url.jpg"
+
+    SafeApp.objects.filter(icon_url__isnull=True).update(icon_url=default_icon_url)
+    SafeApp.objects.filter(icon_url="").update(icon_url=default_icon_url)
```

Comment:
`from django.db.models import Q` at the beginning of the file, then:

```suggestion
    SafeApp.objects.filter(Q(icon_url__isnull=True) | Q(icon_url="")).update(icon_url=default_icon_url)
```